app/services/jokes_api.js

```diff
@@ -0,0 +1,10 @@
+const axios = require('axios');
+
+exports.getJokes = async() =>{
+  try {
+    const data = await axios.get('https://geek-jokes.sameerkumar.website/api?format=json');
```

Comment:
- Change the variable name to something more representative, data is generic. For example:`apiResponse`.
- Put the URI in a constants that could be located in constants path.

⚠️ When we have constants that will be use in other files or just are general constants like URI's , it is better to put them in a constants file.